### ISSUE TITLE
Just gonna say it here.

### ISSUE THREAD

**Issue by ghost** [cosp-project/cosp-project.github.io#1]
So F* Telegram and that unsecure bulls*it. Straight to the point. I downloaded and flashed your ROM on my phone, which made me feel good actually and happy. But after rebooting and seeing MF GOOGLE!!! as the boot loader. Made me feel betrayed and so I reflashed LineageOS right away. And the launcher was bloated with uncustomizable google search widgets and UGH. This ROM probably isn't finished yet with all the removal of stuff but goddamn I can't trust your ROM ever again in the future, tho I might flash it some day again. But probably not for another year.
Is this hurting your feelings? Probably, but delete it for safety measures. You don't want other people seeing this.

**Comment by edi194** [comment]
I'll tell you something, Android is from Google, you shouldn't trust it at all because yk, spying

**Comment by ghost** [comment]
Yes, I know. That's why I bought Librem5. Just wanting to try out your ROM as it seems very cool. But all that other stuff got me pretty good, made me turn the other way

**Comment by SphericalKat** [comment]
@youngbasta Wat

**Comment by edi194** [comment]
So just wait for Q, it'll be rebuilt from scratch without any goolag stuff  ¯\_(ツ)_/¯

**Comment by p8tgames** [comment]
@youngbasta you must be fun at partys

**Comment by RadekBledowski** [comment]
@p8tgames xXXD

**Comment by ghost** [comment]
@edi194 Oh, well when is this going to happen? Like next year? Google is probably releasing Q in Q3 of 2019. So how long will it take you guys then?

**Comment by edi194** [comment]
@youngbasta Idk, I'll know when I'll get Q source 😶

**Comment by FacuM** [comment]
I saw a couple of psychological advisory centers taking people for free, guess you know what to do mate. @youngbasta
